awx/ui/client/features/jobs/index.view.html

```diff
@@ -7,6 +7,68 @@
 
     <div class="col-md-8">
         <at-panel class="at-Stdout">
+            <!-- search ============================================================================================================= -->
+            <form ng-submit="vm.search()">
+            <div class="input-group">
+                <input type="text"
+                       class="form-control at-Input"
```

Comment:
the styling / markup is still a mess and mostly copy-pasted from other places. Will need to factor it down into components